CHANGELOG.md

```diff
@@ -15,6 +15,10 @@
 
         ./bin/rake stack:commands:execute_recipes_on_layers layers="Admin, Workers, Engage" recipes="mh-opsworks-recipes::rsyslog-to-loggly"
 
+* *OPTIONAL EDITS TO THE CLUSTER CONFIG*:
+  Recipes, etc for setting up Analytics layer/node. See README.analytics.md in 
+  mh-opsworks for setup instructions.
```

Comment:
Are you saying those recipes should be re-run even though nothing would have changed about the outcome of running them?